src/Symfony/Component/Form/FormInterface.php

```diff
@@ -103,6 +103,24 @@ public function all();
      */
     public function getErrors($deep = false, $flatten = true);
 
+    /**
+     * Returns the list of the current form and all child forms.
+     *
+     * @return FormErrorIterator An iterator over the {@link FormError}
+     *                           instances that where added to the form
+     *                           and the child forms
+     */
+    public function getAllErrors();
```

Comment:
We cannot change the interface until Symfony 5.0 for BC.

Reply:
I actually totally forgot about that part and agree. I've changed the description.